types/hapi/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for hapi 16.1
+// Type definitions for hapi 16.2
```

Comment:
Please don't bump this version number unless you've also updated the typings to match the hapi 16.2 docs https://hapijs.com/api/16.2.0 and changelog https://hapijs.com/updates